compiler/include/symbol.h

```diff
@@ -676,9 +731,16 @@ void initAstrConsts();
 // pass-by-reference intents are used.
 bool argMustUseCPtr(Type* t);
 
+// Is 'expr' a SymExpr for the outerVar of some ShadowVarSymbol?
+bool isOuterVarOfShadowVar(Expr* expr);
+
+// Verification support.
+void verifyOnFSIntentVarList(DefExpr* def);
```

Comment:
No idea what FSIntentVarList is supposed to mean to me. I'd expect ForallStmt / ShadowVarSymbol to have methods to do any appropriate verification step.

Reply:
Never mind. I will get rid of this function.